scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.adoc

```diff
@@ -18,3 +18,5 @@ include::modules/consuming-huge-pages-resource-using-the-downward-api.adoc[level
 * xref:../nodes/containers/nodes-containers-downward-api.adoc#nodes-containers-downward-api[Allowing containers to consume Downward API objects]
 
 include::modules/configuring-huge-pages.adoc[leveloffset=+1]
+
+include::modules/configuring-transparent-huge-pages.adoc[leveloffset=+1]
```

Comment:
If you do change the name of the file, be sure it matches here. Another reason why renaming is a good idea is because the module above is also called "configuring huge pages" and that could be confusing too.